src/DatePicker/DatePicker.less

```diff
@@ -0,0 +1,87 @@
+@import (reference) "../less/index";
+@import "~react-datepicker/dist/react-datepicker.min.css";
+
+.DatePicker {
+  background-color: @neutral_off_white;
+  position: absolute;
+  margin-top: @size_2xs;
+  .border--s(@neutral_silver);
+  z-index: @zIndex2;
+}
+
+.DatePicker--header {
```

Comment:
nit: only modifiers should be camelCase, these should be PascalCase .DatePicker--Header